OPAL/br/src/main/scala/org/opalj/br/fpcf/analyses/immutability/ClassImmutabilityAnalysis.scala

```diff
@@ -350,33 +318,30 @@ class ClassImmutabilityAnalysis(val project: SomeProject) extends FPCFAnalysis {
 
                 // Field Immutability related dependencies:
                 case FinalP(TransitivelyImmutableField) =>
+
                 case FinalP(NonTransitivelyImmutableField) =>
                     maxLocalImmutability = NonTransitivelyImmutableClass
-                case FinalP(MutableField) => return Result(t, MutableClass);
-                case UBP(MutableField)    => return Result(t, MutableClass);
-                case ELBP(e, NonTransitivelyImmutableField |
-                    TransitivelyImmutableField) => dependees -= e
-                case UBP(TransitivelyImmutableField)    => // no information about field mutability
-                case UBP(NonTransitivelyImmutableField) => maxLocalImmutability = NonTransitivelyImmutableClass
+
+                case FinalP(MutableField)                                            => return Result(t, MutableClass);
+                case UBP(MutableField)                                               => return Result(t, MutableClass);
+
+                case LBP(NonTransitivelyImmutableField | TransitivelyImmutableField) =>
```

Comment:
This case now does not have an effect and thus needs a comment explaining why it does nothing (just like below)